src/test/fixtures/profiles/make-profile.js

```diff
@@ -294,7 +312,8 @@ function _parseTextSamples(textSamples: string): Array<string[]> {
 
 function _buildThreadFromTextOnlyStacks(
   textOnlyStacks: Array<string[]>,
-  funcNames: string[]
+  funcs: Array<{| funcName: string, category: number | null |}>,
```

Comment:
nit: Instead of number, should it be IndexToCategoryList ?

Reply:
Yes it should. But this comment no longer applies because I've changed the argument back to `funcNames` during the rebase, so there's nothing to address here.